ansible/roles/romanesco-variables/defaults/main.yml

```diff
@@ -10,7 +10,6 @@
     romanesco_net_interface: eth0
 
     romanesco_main_app_name: romanesco
```

Comment:
@aashish24 I am just curious what romanesco is, I think it's the old name for girder worker?

Reply:
@aashish24 would know.